tests/unit/src/main/java/org/mule/tck/SimpleUnitTestSupportScheduler.java

```diff
@@ -38,7 +38,7 @@ public SimpleUnitTestSupportScheduler(int corePoolSize,
 
   @Override
   public void stop(long gracefulShutdownTimeoutSecs, TimeUnit unit) {
-    // Nothing to do. The lifecycle of this pool is managed by the UnitTestSchedulerService that instantiated this
+    shutdownNow();
```

Comment:
you used to be fine with dong nothing. Won't there be any problems if you shutdown twice (once here and another in the owning lifecycle)?